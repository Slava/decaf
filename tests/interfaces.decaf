interface Runnable {
  void run();
  void stop(int x);
}

