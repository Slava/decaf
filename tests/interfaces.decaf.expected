Program
  Interface Runnable
    void: run:
    void: stop: <int x>
